src/types/index.ts

```diff
@@ -443,11 +440,13 @@ export type ICommonTransaction = ITransaction | IActiveMultisigTransaction
 
 export type ITransactionsState = {
   loaded: ITransaction[];
-  nextPageUrl?: string;
+  nextPageUrl: string | null;
   pending: Record<string, ITransaction[]>;
   tipWithdrawnTransactions: ITransaction[];
 }
 
+export type ITransactionStateList = Record<Encoded.AccountAddress, ITransactionsState>
```

Comment:
I suggest to rename this to `IAccountTransactionList` in order to make it more meaningfull.